.travis.yml

```diff
@@ -79,6 +79,7 @@ jobs:
         - bundle exec bundle-audit check --update
         - bin/test-console-check
         - yarn build-storybook
+        - yarn cypress install
```

Comment:
this cost (to download and install  cypress) is only paid in the one job - seems like a benefit (the rspec jobs don't install or carry cypress).

Reply:
Cypress tests are not the fastest, how will fetching an 800 MB before the build influence overall build times?